pkg/innerring/processors/container/process_eacl.go

```diff
@@ -42,12 +42,11 @@ func (cp *Processor) checkSetEACL(e container.SetEACL) error {
 	binTable := e.Table()
 	tableHash := sha256.Sum256(binTable)
 
+	// verify the identity of the container owner
```

Comment:
Previous position was correct, `key.Verify` just check the signature as said above.

Reply:
Fixed